clustering/marathon.py

```diff
@@ -0,0 +1,555 @@
+     - The amount of disk in MB that is reserved for the application per instance.
+
+  ports:
+    required: false
+    description:
+     - An array of required port resources on the host.
+
+  requirePorts:
+    required: false
+    description:
+     - If true, the ports you have specified are used as host ports.
+
+  instances:
+    required: false
+    description:
+     - The number of instances of this application to start.
+
+  executor:
+    required: false
+    description:
+     - The executor to use to launch this application.
+
+  user:
+    required: false
+    description:
+      - The user to use to launch this application.
+
+  container:
+    required: false
+    description:
+     - Additional data passed to the containerizer on application launch. This is a free-form data structure that can contain arbitrary data.
+
+  docker_image:
+    required: false
+    description:
+     - Name of the Docker image. Ignored if container is defined.
+
+  docker_forcePullImage:
+    required: false
+    description:
+     - Force Docker to pull the image before launching each task. Ignored if container is defined.
+
+  docker_privileged:
+    required: false
+    description:
+     - Allows users to run containers in privileged mode. Ignored if container is defined.
+
+  docker_network:
+    required: false
+    description:
+     - Type of networking for the Docker container. Ignored if container is defined.
+
+  docker_portMappings:
+    required: false
+    description:
+     - Port mappings for the Docker container. Ignored if container is defined.
+
+  docker_parameters:
+    required: false
+    description:
+     - Arbitrary parameters for the Docker container. Ignored if container is defined.
+
+  container_volumes:
+    required: false
+    description:
+     - Array of volumes for the container. Ignored if container is defined.
+
+  env:
+    required: false
+    description:
+     - Key value pairs that get added to the environment variables of the process to start.
+
+  constraints:
+    required: false
+    description:
+     - Valid constraint operators are one of ["UNIQUE", "CLUSTER", "GROUP_BY"].
+
+  acceptedResourceRoles:
+    required: false
+    description:
+     - A list of resource roles.
+
+  labels:
+    required: false
+    description:
+     - Attaching metadata to apps can be useful to expose additional information to other services, so we added the ability to place labels on apps.
+
+  uris:
+    required: false
+    description:
+     - URIs defined here are resolved, before the application gets started. If the application has external dependencies, they should be defined here.
+
+  storeUrls:
+    required: false
+    description:
+     - a sequence of URIs, that get fetched on each instance, that gets started. The artifact could be fetched directly from the source, or put into the artifact store. One simple way to do this is automatic artifact storing.
+
+  dependencies:
+    required: false
```

Comment:
if required: false is defined you must define a default e.g. default: null also see http://docs.ansible.com/ansible/developing_modules.html#module-checklist